contributors/design-proposals/architecture/architecture.md

```diff
@@ -170,7 +170,7 @@ Kubernetes supports user-provided schedulers and multiple concurrent cluster sch
 using the shared-state approach pioneered by 
 [Omega](https://research.google.com/pubs/pub41684.html). In addition to the disadvantages of
 pessimistic concurrency described by the Omega paper, 
-[two-level scheduling models](http://mesos.berkeley.edu/mesos_tech_report.pdf) that hide information from the upper-level
+[two-level scheduling models](https://people.eecs.berkeley.edu/~alig/papers/mesos.pdf) that hide information from the upper-level
```

Comment:
https://amplab.cs.berkeley.edu/wp-content/uploads/2011/06/Mesos-A-Platform-for-Fine-Grained-Resource-Sharing-in-the-Data-Center.pdf looks like the official dated version

Reply:
I'm ok with both. :)

Reply:
If it's necessary I could change the link to the one suggested.

Reply:
Please amend the commit with this URL. Thank you!